### ISSUE TITLE
Tenant Jenkins needs to provide more insights into "Build" data

### ISSUE THREAD

**Issue by pradeepto** [openshiftio/openshift.io#2245]
Right now, we have no visibility into what happens in the tenant Jenkins. How many builds fail for OSIO user? How many builds pass? How long does a build take?

We need to define what are the data points we need to measure and track. All these data need to be fed into Zabbix.

There has been some work towards monitoring Jenkins albeit it was for monitoring the fabric8cd Jenkins instance itself. Maybe some of the work can be reused.

- [ ] Create an initial list of data points that need to be measured.
- [ ] Update tenant Jenkins image with monitoring endpoints/agents that can send required data points to Zabbix.

Reference issues : 

- https://github.com/fabric8io/fabric8-build-team/issues/4
- https://github.com/fabric8io/fabric8-build-team/issues/24
- https://github.com/fabric8-services/fabric8-tenant-jenkins/pull/57

**Comment by chmouel** [comment]
@sthaha I was wondering if you had an update on this?

**Comment by sthaha** [comment]
@chmouel no, I only made minor progress with enabling the timestamp but then this task is still in "Backlog" and there was another one that was in progress which got assigned to me - #2403 so started work on that.
